v/vert.x/eclipse_vert_x_build_ppc64le_ubi8.4.sh

```diff
@@ -0,0 +1,125 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : vert.x
+# Version       : 3.9.7 and 4.1.2
+# Source repo   : https://github.com/eclipse-vertx/vert.x
+# Tested on     : rhel UBI 8.4
+# Script License: Apache License, Version 2 or later
+# Maintainer's  : Santosh Magdum <santosh.magdum@us.ibm.com>
+#                 Priya Seth <priya.seth@us.ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+#!/bin/bash
+
+PACKAGE_NAME=vert.x
+PACKAGE_PATH=github.com/eclipse-vertx/vert.x
+PACKAGE_VERSION=${1:-3.9.7}
+PACKAGE_URL=https://github.com/eclipse-vertx/vert.x
+
+BUILD_HOME=`pwd`
+
+echo "`date +'%d-%m-%Y %T'` - Staring eclipse vert.x build. Dependencies will be cloned in $BUILD_HOME"
+
+# ------- Install dependencies -------
+
+yum -y install maven make python3
+
+yum -y install subscription-manager.ppc64le
+yum -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+subscription-manager repos --enable rhel-7-for-power-le-extras-rpms
+yum -y install epel-release
+
+yum -y install gcc-c++.ppc64le
+yum -y install wget
+yum -y install git
+yum -y install java-1.8.0-openjdk-devel
+dnf install openssl-devel.ppc64le -y
```

Comment:
any reason for using a mix of yum/dnf? and maybe we can combine into a single install command?